packages/aws/src/application/AWSAccount.ts

```diff
@@ -199,7 +199,7 @@ export default class AWSAccount extends CloudProviderAccount {
       new CloudWatch(options),
       new CloudWatchLogs(options),
       new CostExplorer({
-        region: 'us-east-1',
+        region: configLoader().AWS.IS_AWS_GLOBAL?'us-east-1':'cn-northwest-1',
```

Comment:
I think rather than a new config called `IS_AWS_GLOBAL`, this could be named `COST_EXPLORER_REGION` where you can specify a preferred region and it would default to `us-east-1`

Reply:
I believe based on your latest comment, we should be able to update this to be `configLoader().AWS.ATHENA_REGION` and update it to default to `us-east-1`